packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -536,7 +537,7 @@ export interface DatabaseInstanceNewProps {
    * The CloudFormation policy to apply when the instance is removed from the
    * stack or replaced during an update.
    *
-   * @default RemovalPolicy.Retain
+   * @default Snapshot (remove the resource, but retain a snapshot of the data)
```

Comment:
Feels right to add `Snapshot` to the `RemovalPolicy` enum.

Any specific reason you avoided that?

Reply:
Nope. Done.